server/mux.go

```diff
@@ -127,46 +127,40 @@ func (m *Mux) mustAddRoute(root *pathpattern.Node, methods []string, path string
 			}},
 		}
 	}
+
 	return m
 }
 
 func (m *Mux) FindHandler(req *http.Request) http.Handler {
 	var route *openapi3filter.Route
 
-	node, srvCtxOpts, paramValues := m.match(m.endpointRoot, req)
+	if m.endpointRoot == nil {
```

Comment:
All `Mux` objects created with `NewMux` will have a non nil endpointRoot. 
This check is obsolete here and should be tested on configload. The question is the expectation... suffixList length? A server without endpoints is still valid with file or spa root.

Reply:
Done